shark/iree_utils.py

```diff
@@ -135,12 +135,18 @@ def get_iree_frontend_args(frontend):
 def get_iree_module(module, device, input_type, args, func_name):
     flatbuffer_blob = None
     # Compile according to the input type, else just try compiling.
-    if input_type in ["mhlo", "tosa"]:
+    if input_type in ["tosa"]:
         flatbuffer_blob = ireec.compile_str(
             str(module),
             target_backends=[IREE_DEVICE_MAP[device]],
             extra_args=args,
             input_type=input_type)
+    elif input_type in ["mhlo"]:
+        flatbuffer_blob = ireec.compile_str(
+            module,
+            target_backends=[IREE_DEVICE_MAP[device]],
+            extra_args=args,
+            input_type=input_type)
```

Comment:
can we just add if input_type = tosa: module = str(module)?

Reply:
Actually, it's fine for now, I need to also push up a change that has conflict with this, will clean up there :)

Reply:
+2 for push and iterate